pkg/getter/plugingetter_test.go

```diff
@@ -49,53 +47,53 @@ func TestCollectPlugins(t *testing.T) {
 	}
 }
 
-func TestPluginGetter(t *testing.T) {
-	if runtime.GOOS == "windows" {
-		t.Skip("TODO: refactor this test to work on windows")
-	}
-
-	env := cli.New()
-	env.PluginsDirectory = pluginDir
-	pg := NewPluginGetter("echo", env, "test", ".")
-	g, err := pg()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	data, err := g.Get("test://foo/bar")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	expect := "test://foo/bar"
-	got := strings.TrimSpace(data.String())
-	if got != expect {
-		t.Errorf("Expected %q, got %q", expect, got)
-	}
-}
-
-func TestPluginSubCommands(t *testing.T) {
-	if runtime.GOOS == "windows" {
-		t.Skip("TODO: refactor this test to work on windows")
-	}
-
-	env := cli.New()
-	env.PluginsDirectory = pluginDir
-
-	pg := NewPluginGetter("echo -n", env, "test", ".")
-	g, err := pg()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	data, err := g.Get("test://foo/bar")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	expect := "   test://foo/bar"
-	got := data.String()
-	if got != expect {
-		t.Errorf("Expected %q, got %q", expect, got)
-	}
-}
+// func TestPluginGetter(t *testing.T) {
```

Comment:
i get that you want to fix post render plugins with the updated interfaces in a separate PR, but we should get this one working in this PR, yes?

Reply:
(test was added)